cmd/k8s-operator/proxy.go

```diff
@@ -33,7 +33,16 @@ import (
 
 var whoIsKey = ctxkey.New("", (*apitype.WhoIsResponse)(nil))
 
-var counterNumRequestsProxied = clientmetric.NewCounter("k8s_auth_proxy_requests_proxied")
+var (
+	// counterNumRequestsproxies counts the number of API server requests proxied via this proxy.
+	counterNumRequestsProxied = clientmetric.NewCounter("k8s_auth_proxy_requests_proxied")
+
+	// counterSessionRecordingsAttempted counts the number of session recording attempts.
+	counterSessionRecordingsAttempted = clientmetric.NewCounter("k8s_session_recordings_attempted")
```

Comment:
Should these maybe keep the same prefix as the above, e.g. `k8s_auth_proxy_session_recordings_attempted`, if that's not too long?

Reply:
Thanks, fixed